src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names.mdx

```diff
@@ -457,13 +457,27 @@ Not every field listed in this glossary is required for every condition type. Th
 
     In the UI, under **Advanced signal settings**, this is the **Aggregation window** field.
 
-    Default is **60 seconds**. Maximum is 15 minutes.
+    Default is **60 seconds**. Maximum is 120 minutes.
 
     Used for:
     * NRQL conditions
 
   </Collapser>
 
+  <Collapser
+    id="signal_slideby"
+    title="signal[slideby]"
+  >
+    By default, aggregation windows are grouped sequentially. This can lead to spiky charts whenever one window starts and another begins.
+
+    Use `slideby` to create sliding windows. Sliding aggregated windows overlap, creating smoother charts. The `slideby` interval sets the duration of the overlap.
+
+    In the UI, under **Advanced signal settings**, this is the **Use sliding window aggregation** toggle.
+
+    Default is based on the current window duration. The slide by interval must divide evenly into your aggregation window duration. 
```

Comment:
Potentially also add that slide by must be less than the aggregation window duration